__tests__/app.test.js

```diff
@@ -92,3 +93,38 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: Responds with an array of article objects with the correct properties and sorted by date (descending)", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toHaveLength(13);
+
+        body.article.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+              comment_count: expect.any(Number),
+            })
+          );
+        });
+        expect(body.article).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("404: Responds with an error if the endpoint is incorrect", () => {
```

Comment:
this isn't related to the /api/articles endpoint - so it should get it's own describe!